doc/api/v1.yaml

```diff
@@ -275,6 +275,40 @@ paths:
         '404':
           description: The requested vote plan does not exist or is not active.
 
+  /api/v1/votes/plan/account-votes/{account_id}:
+    get:
+      description: Get numbers of proposals a given user have voted related to the each vote plan.
```

Comment:
nit: I think something like 'Get indexes of proposals a given user has voted for across all active vote plans'  would read better